### PR TITLE
Single product page and fix routing

### PR DESCRIPTION
**Pull request by Dev-AkshatGupta**
Please check only majory three files that are 
[authSlice](https://github.com/Dev-AkshatGupta/Piston-Parts-Ecommerce/blob/dbde50a55ad875b69c870a6d02568efe8a93b4f3/src/AllRoutes.jsx)
[src/AllRoutes.jsx](https://github.com/Dev-AkshatGupta/Piston-Parts-Ecommerce/blob/dbde50a55ad875b69c870a6d02568efe8a93b4f3/src/AllRoutes.jsx)

[OperationsSlice](https://github.com/Dev-AkshatGupta/Piston-Parts-Ecommerce/blob/dbde50a55ad875b69c870a6d02568efe8a93b4f3/src/Redux/Reducers-Redux/operationsSlice.js)

Please ignore any file in the back end that's need not to be reviewed
Thanks for reviewing this pr

### CODE REVIEW

File: src/Redux/Reducers-Redux/operationsSlice.js [hunk 1]
```diff
@@ -0,0 +1,369 @@
...
+      console.log(error);
+    }
+  }
+);
+
+export const getAllAddress = createAsyncThunk(
+  "operations/getAllAddress",
+  async (_, { rejectWithValue }) => {
+    try {
+      // console.log(details);
```
Comment (Hamza-777) [inline src/Redux/Reducers-Redux/operationsSlice.js:56]: remove this commented console log

File: src/Redux/Reducers-Redux/operationsSlice.js [hunk 2]
```diff
@@ -0,0 +1,369 @@
...
+       const {_id: id } = details;
+       const { data } = await axios.post(`/api/user/address/${id}`,{
+         address:details
+       }, {
+         headers: {
+           authorization: encodedToken,
+         },
+       });
+       notifyWarn("Address Removed");
+       console.log(data);
```
Comment (Hamza-777) [inline src/Redux/Reducers-Redux/operationsSlice.js:123]: this one too

**Review by Hamza-777 — APPROVED** [review]
Nice work amazing as always

File: src/AllRoutes.jsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
+import { ProductsPage } from "./pages/products-page/ProductsPage";
```
Comment (mikesch803) [inline src/AllRoutes.jsx:1]: suggestion : you would have an index file, just import all pages from it

**Review by mikesch803 — APPROVED** [review]
nothing much to suggest, good to merge

File: src/AllRoutes.jsx (hunk 3 above)
Comment (Dev-AkshatGupta) [reply src/AllRoutes.jsx:1]: I will do it in next pr Thanks for the suggestion

**Pull request merged**